### DISCUSSION TITLE
Curator integration

### DISCUSSION THREAD

**Discussion by ch314s** [awcodes/filament-tiptap-editor#299] [Q&A]
How to integrate with https://github.com/awcodes/filament-curator ?

**Comment by awcodes** [comment]
Publish the Tiptap Editor config and change the MediaAction class. It's commented in the config file.

**Reply by mohsen-yousefi** [reply]
i have this error when i change MediaAction  
Method FilamentTiptapEditor\TiptapEditor::getImageCropAspectRatio does not exist.

**Comment by awcodes** [comment]
make sure you are on the latest version of both packages.
